encodings/alp/src/alp/mod.rs

```diff
@@ -19,7 +26,14 @@ impl Display for Exponents {
     }
 }
 
-pub trait ALPFloat: Float + Display + 'static {
+mod private {
```

Comment:
in theory this was previously extensible, but we probably want to constrain it

Reply:
poor f16 not considered in the paper. Anyway this is the right thing to do

Reply:
ah i too forgot about f16. I suppose that we probably want special compressors for things like bf16

Reply:
Yes, this is just a comment for future readers. The paper only talked about the common float types